policyengine_us/parameters/gov/states/wv/tax/income/credits/low_income_family_tax_credit/income_threshold.yaml

```
@@ -0,0 +1,28 @@
+description: West Virginia qualifies filers for the low-income family tax credit with modified adjusted gross income at or below this threshold, based on filing status.
```

Delete the file but for reference this would be the appropriate description (it read as if the income limit was $2,700 total)
```suggestion
description: West Virginia limits its low-income family tax credit to filers with modified adjusted gross income at or below a fraction of the federal poverty guideline plus this threshold, based on filing status.
```